.gitignore

```diff
@@ -1 +1,2 @@
-kubectl-gs
+kubectl-gs*
```

Comment:
What does the star mean here?

Reply:
It means that it will ignore all files that begin with the `kubectl-gs` prefix.

When running, `make build-darwin`, you can actually have multiple versions of the executable side by side (in the format of `kubectl-gs-{TAG}-{OS}-{ARCH}`), so this helps us not include all of those in the repo